### PR TITLE
Update nest media attachment documentation

### PR DESCRIPTION
**Pull request by allenporter**
## Proposed change

Update nest media attachment documentation. This improves documentation to further clarify the role of event entities vs device triggers and updates the documentation for use with attachments.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/124554 and https://github.com/home-assistant/core/pull/123042
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Enhanced documentation on camera integrations, detailing event entities for automation and immediate event notifications.
	- Introduction of an “attachment” field in device trigger payloads for dynamic media notifications.
- **Documentation**
	- Updated terminology in feature summaries for clarity, such as changing "Triggers / Events" to "Event Entities / Triggers."
	- Improved explanations of how Nest event entities and media attachments function within the Home Assistant ecosystem.

### CODE REVIEW

File: source/_integrations/nest.markdown [hunk 1]
```diff
@@ -263,6 +263,8 @@ Home Assistant supports all SDM API features. However, every Camera or Doorbell
 - **RTSP**: These devices have an HLS stream served by the Home Assistant Core. These cameras support server-side `camera` actions like stream recording or image preview. See [Low Latency HLS](/integrations/stream#ll-hls) as a great option to enable to reduce stream latency.
 - **WebRTC**: These devices support direct browser to camera communication and a super low latency stream. A [Picture Glance Card](/dashboards/picture-glance/) can show the live stream in the grid with the *Camera View* set to `live` (not recommended for battery-powered cameras). `camera` actions like stream recording are *not supported*.
 
+Given a camera named `Front Yard` then the camera is created with a name such as `camera.front_yard`.
```
Comment (c0ffeeca7) [inline source/_integrations/nest.markdown]:
```suggestion
Given a camera named `Front Yard`, then the camera is created with a name such as `camera.front_yard`.
```

File: source/_integrations/nest.markdown [hunk 2]
```diff
@@ -272,10 +274,10 @@ This feature is enabled by the following permissions:
 
 {% endnote %}
 
+All cameras also expose event entities for automation. Some camera models also
+support capturing media (snapshots or clips) through device triggers. The table below summarizes the [Supported SDM API features](https://developers.google.com/nest/device-access/supported-devices) for each device.
```
Comment (c0ffeeca7) [inline source/_integrations/nest.markdown]:
```suggestion
support capturing media (snapshots or clips) through device triggers. The table below summarizes the [supported SDM API features](https://developers.google.com/nest/device-access/supported-devices) for each device.
```

File: source/_integrations/nest.markdown [hunk 3]
```diff
@@ -272,10 +274,10 @@ This feature is enabled by the following permissions:
...
 
-All cameras have motion and person triggers, however only some support capturing snapshots for events. The table below summarizes the [Supported SDM API features](https://developers.google.com/nest/device-access/supported-devices) for each device.
-
-| Device                                                                           |    Live Stream    |         Triggers / Events          | Media Source<br> for Triggers / Events |
+| Device                                                                           |    Live Stream    |         Event Entities / Triggers          | Media Source<br> for Triggers |
```
Comment (c0ffeeca7) [inline source/_integrations/nest.markdown]:
```suggestion
| Device                                                                           |    Live stream    |         Event entities / triggers          | Media source<br> for triggers |
```

File: source/_integrations/nest.markdown [hunk 4]
```diff
@@ -321,13 +323,18 @@ This is an example of what the `nest_event` payload looks like for a Device Trig
...
+        }
         "zones": ["Zone 1"],
     },
 }
 ```
 
 - `device_id`: The Home Assistant device identifier for the camera
-- `nest_event_id`: is an opaque identifier that can be used with the Media Source Attachments described below for supported cameras.
+- `nest_event_id`: is an opaque identifier that identifies the event.
+- `attachment`: May be present if the device supports snapshots or clips and depends on the capabilities of the device. This is a URL where media from the media source can be fetched.
```
Comment (c0ffeeca7) [inline source/_integrations/nest.markdown]:
```suggestion
- `attachment`: May be present if the device supports snapshots or clips and depends on the device's capabilities. This is a URL where media can be fetched from the media source.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @allenporter :+1:

**Comment by allenporter** [comment]
> Thank you, @allenporter 👍

Thank you for the high quality review.  I appreciate all the wording improvements and consistency fixes.

**Pull request merged**